src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Shape {
```

Comment:
all our figures should implement interfaces `Drawable` and `AreaCalculator` - how can we optimise the code so that we won't have to write `implements Drawable, AreaCalculator` for Circle and other child figures?

Reply:
done